### ISSUE TITLE
[BUG] Search: Missing german translations in type selection

### ISSUE THREAD

**Issue by hurradieweltgehtunter** [owncloud/ios-app#1211]
### Steps to reproduce
Set app to german.

1. Tap on search bar
2. Tap on "Typ" for type selection

### Expected behavior
Shown list of types is translated

### Actual behavior
![IMG_B5D2DCEFB2FD-1](https://user-images.githubusercontent.com/16665512/236431068-893f740a-7045-4eb4-8e1d-23fb8e4c4e16.jpeg)

ownCloud  Beta version 12.0 build 258
(app: 9f741369, sdk: aeb1516)

**Comment by michaelstingl** [comment]
Still an issue:

```
ownCloud  Beta version 12.0 build 260 (Fri May  5 17:11:28 CEST 2023)
(app: v12.0-beta.1-2-g00337b74 - fix/v12-fixes@00337b74, sdk: v12.0-beta.1-1-ga7f04ed - feature/graph-api@a7f04ed)
```

**Comment by felix-schwarz** [comment]
Fixed in #1207:
![Simulator Screenshot - iPad (10th generation) - 2023-05-09 at 10 22 33](https://user-images.githubusercontent.com/639669/237038123-e59843be-6ed7-43de-b71e-45651dbc7cf2.png)
